src/frontends/arcs/particle_spec_test.cc

```diff
@@ -33,46 +32,42 @@ struct ParticleSpecProtoAndExpectedInfo {
   std::vector<Edge> expected_edges;
 };
 
-class ParticleSpecFromProtoTest :
-   public testing::TestWithParam<ParticleSpecProtoAndExpectedInfo> {
+class ParticleSpecFromProtoTest
+    : public testing::TestWithParam<ParticleSpecProtoAndExpectedInfo> {
  protected:
   ParticleSpecFromProtoTest()
       : particle_spec_(CreateParticleSpec(type_factory_)) {}
 
   static std::unique_ptr<ParticleSpec> CreateParticleSpec(
-      types::TypeFactory& type_factory) {
-    arcs::ParticleSpecProto particle_spec_proto;
+      ir::types::TypeFactory& type_factory) {
+    ::arcs::ParticleSpecProto particle_spec_proto;
     std::string textproto = GetParam().textproto;
-    CHECK(google::protobuf::TextFormat::ParseFromString(
-        textproto, &particle_spec_proto))
+    CHECK(google::protobuf::TextFormat::ParseFromString(textproto,
+                                                        &particle_spec_proto))
         << "Particle spec textproto did not parse correctly.";
     return proto::Decode(type_factory, particle_spec_proto);
   }
 
-  types::TypeFactory type_factory_;
+  ir::types::TypeFactory type_factory_;
   std::unique_ptr<ParticleSpec> particle_spec_;
 };
 
 // Locally defined operator== for use in test below.
-bool operator ==(const ir::TagCheck& check1,
-                 const ir::TagCheck* check2) {
+bool operator==(const TagCheck& check1, const TagCheck* check2) {
```

Comment:
Perhaps this should be `static` to prevent it being accidentally used elsewhere?

Reply:
done. Actually, it is not a good idea to define `operator==` outside of the class. Just made this static for now.